### PR TITLE
Adding Pipfile and Pipfile.lock for specifying the requirements/packages

### PR DESCRIPTION
**Pull request by suppathak**
Along with my blog post, I also want to include  Pipfile and Pipfile.lock in order to give the information about the required packages for the analysis.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+sklearn = "*"
```
Comment (chauhankaranraj) [inline Pipfile]:
```suggestion
scikit-learn = "*"
```
According to the pypi docs - https://pypi.org/project/sklearn/

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]

**Review by tumido — APPROVED** [review]
LGTM :+1:

**Comment by chauhankaranraj** [comment]
Thanks @suppathak, LGTM! Could you please squash commits?

**Comment by suppathak** [comment]
> Thanks @suppathak, LGTM! Could you please squash commits?

Done! Thanks :)

**Review by chauhankaranraj — APPROVED** [review]

**Pull request merged**